packages/components/src/public/QueryModel/withQueryModels.tsx

```diff
@@ -56,6 +56,55 @@ export function withSearchParams<T>(Component: WithSearchParamsComponent<T>): Co
     return Wrapped;
 }
 
+function columnHasFilter(fieldKey, filters: Filter.IFilter[]): boolean {
+    return filters.some(filter => filter.getColumnName() === fieldKey);
+}
+
+function columnsHaveFilter(columns: string[], filters: Filter.IFilter[]): boolean {
```

Comment:
minor: with all of the column name vs fieldKey stuff, I might suggest that this first param be changed from "columns" to something like "columnFieldKeys"

Reply:
great idea